### DISCUSSION TITLE
Error: Seb failed to start a new session!

### DISCUSSION THREAD

**Discussion by nzwice** [SafeExamBrowser/seb-win-refactoring#225] [General]
Hi, I installed seb using Windows 10, the installation was fine, but when I open the app, I got an error like this:
![244360907_1212538015909076_4816368383801992919_n](https://user-images.githubusercontent.com/49333883/136709164-5dd3e284-7364-40f7-9f1d-cbb30a90af8d.png)
I've already uninstalled all antivirus software, turn off windows security but still does not work.
Here is the log files:
[2021-10-11_01h16m27s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7318004/2021-10-11_01h16m27s_Runtime.log)
[2021-10-11_01h16m27s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7318005/2021-10-11_01h16m27s_Client.log)
How to fix that?

**Comment by dbuechel** [comment]
It appears that the application components of SEB can't communicate with each other on your system. Are you using any security / anti-virus software which could block inter-process communication? SEB uses WCF (Windows Communication Foundation), so make sure that the .NET Framework and WCF in particular work on your system.

**Reply by Donzelll** [reply]
![image](https://github.com/user-attachments/assets/b7231475-9dc2-4b2f-865b-be03a78c6a7b)
Hi i have the same problem and picture above is my log. Any advice?

**Reply by dbuechel** [reply]
Use the Reset Utility to fix the ease of access configuration.

**Comment by mnsosa** [comment]
Any solution? I have the same problem.

**Reply by dbuechel** [reply]
The error message above is generic and may have many causes. Do you have the same error as in the logs attached above or a different one? In either case, please post your log files here so that we may investigate the issue further.

**Comment by ciccio1404** [comment]
Salve, ho lo stesso identico problema... Chiedendomi di immettere 'password delle impostazioni'. Ha funzionato nei giorni scorsi, ed oggi ha fatto così. Tra 7 giorni devo sostenere il Tolc, come faccio?

**Comment by arvemell** [comment]
We have the exact issue here. "ERROR: Failed to connect to the runtime. Aborting startup..." There has been about 10-20 students so far with this issue. Tried to reinstall VC++\net frame but still not working :(
Any suggestions?

**Reply by dbuechel** [reply]
Which OS version and build are you using? Could you please provide the log files of the affected sessions and furthermore check the Windows Event logs whether there might be any errors or warnings related to SEB?

**Reply by arvemell** [reply]
[STUDENT 2 2021-11-03_09h16m43s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7475187/STUDENT.2.2021-11-03_09h16m43s_Runtime.log)
[STUDENT 2 2021-11-03_09h16m43s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7475188/STUDENT.2.2021-11-03_09h16m43s_Client.log)
[STUDENT 1 2021-10-11_01h16m27s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7475189/STUDENT.1.2021-10-11_01h16m27s_Client.log)
[STUDENT 1 2021-10-11_01h16m27s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7475190/STUDENT.1.2021-10-11_01h16m27s_Runtime.log)

Would you check under "Windows logs -> Application" in the event viewer? Will remote assist one of them tomorrow.

**Reply by dbuechel** [reply]
Excellent, that would be great, thanks a lot. Yes exactly, these entries would be found in the "Application" event log of Windows. Specifically, there should be entries around the time SEB gets started resp. when the establishing of the component communication fails, e.g. around 2021-11-03 09:16:50 for the first client log you posted above:

```
2021-11-03 09:16:50.529 [01] - WARNING: [RuntimeProxy] Communication channel has faulted!
2021-11-03 09:16:50.554 [01] - WARNING: [RuntimeProxy] Endpoint 'net.pipe://localhost/safeexambrowser/runtime/78994f89-c736-44a0-b256-3364fdc47ab7' could not be found!
2021-11-03 09:16:50.555 [01] - ERROR: Failed to connect to the runtime. Aborting startup...
```

**Reply by arvemell** [reply]
Great, thanks! Will check it out tomorrow.

**Reply by dbuechel** [reply]
It may be that there furthermore are entries in the "System" and/or "Security" event logs, so it would be absolutely awesome if you'd have the time to also potentially check those two logs.

**Reply by arvemell** [reply]
Willdo!

**Reply by arvemell** [reply]
Found out that a lindgys software updater was the issue! Lingdys is used by students with dyslexia. Lingdys is owned by Lingit if im not mistaken and the service in windows is called "Lingit.Updater" (Lingit Updater Service). Attached is a screenshot. When I force close this it works
![lingdys](https://user-images.githubusercontent.com/93716214/140486885-aca74d9b-eae3-4ef6-ba9a-05b4773f780d.PNG)
!

**Reply by dbuechel** [reply]
Thanks a lot for your investigation effort! I'm glad you've found a workaround for the issue, though it appears very strange that this updater service would interfere with the communication between SEB's application components. If you happen to speak Norwegian and if you'd have the time, it would be great if you could let that company know about the incompatibility, as I reckon that quite a few students will be running in the same issue.

**Reply by olegfeferman** [reply]
@arvemell, thanks for finding this issue! We released a hotfix for Lingit.Updater service which fixes the named pipes conflict. If you restart the service (or on next system restart) - it should download and install a new update (version 3.0.53.0). If you still have some issues - let me know.

**Reply by dbuechel** [reply]
@olegfeferman Excellent news! Just out of curiosity: Would you mind sharing what the exact name conflict was which occurred with your updater service? I thought that I chose the names for the pipes between SEB's components sufficiently careful, but I might have missed something and do need to improve the naming in SEB as well.

**Reply by olegfeferman** [reply]
@dbuechel For historical reasons we've used base addresses in ServiceHost. So from my point of view you have nothing to fix.

**Reply by dbuechel** [reply]
@olegfeferman Thanks a lot for your reply. Would you mind elaborating a bit more with regard to the base address you mentioned resp. how that would break the IPC of SEB's components? We had similar issues with SEB 2.x because that version actually used a generic base address, but I tried to remedy this by using a specific base address in version 3.x (namely `net.pipe://localhost/safeexambrowser/...`).

Is it the case that you used `net.pipe://localhost/` as base address and this had the effect of blocking the IPC of SEB? I don't quite understand the root cause of the issue yet and have furthermore seen that these errors are happening quite a bit these days. Thus it would be awesome if you could help me understand the exact root cause so that I can help other users better when they run into the same issue due to another software.

**Reply by olegfeferman** [reply]
@dbuechel we used `net.pipe://localhost/` as base address and then our own name when doing `AddServiceEndpoint`.

**Reply by dbuechel** [reply]
Hm, okay, thanks for the clarification. So in that case it appears that this blocks any other application from registering a ServiceHost for `net.pipe:localhost`, even though the actual path may be different. Not good, but that would explain why there are other support requests with that error since I reckon that there are quite a few applications which (erroneously) use that global base address.

**Comment by EliaSchmid** [comment]
Hello, I have the same problem.
I need to use the safe exam browser for my exams and am the only one with this problem.
Any fixes?

**Reply by dbuechel** [reply]
Could you please post the log files of the affected session?

**Reply by EliaSchmid** [reply]
Sure, Where / How?
I have never used GitHub before...

**Reply by dbuechel** [reply]
You can find the log files on your filesystem under path `%LocalAppData%\SafeExamBrowser\Logs`, and then simply post them here as a comment.

**Reply by EliaSchmid** [reply]
[2023-02-02_15h43m37s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/10686496/2023-02-02_15h43m37s_Runtime.log)
[2023-02-02_15h43m37s_Service.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/10686499/2023-02-02_15h43m37s_Service.log)

Here they are...

**Reply by dbuechel** [reply]
In your case, the reason is that the SEB Service component isn't able to complete its operations:

```
2023-02-02 15:43:40.738 [11] - ERROR: [EaseOfAccessConfiguration] Failed to set registry item 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\Utilman.exe\Debugger' => 'SebDummy.exe'!

   Exception Message: Es wurde versucht, einen nicht autorisierten Vorgang auszuführen.
   Exception Type: System.UnauthorizedAccessException

   bei Microsoft.Win32.RegistryKey.Win32Error(Int32 errorCode, String str)
   bei Microsoft.Win32.RegistryKey.SetValue(String name, Object value, RegistryValueKind valueKind)
   bei Microsoft.Win32.Registry.SetValue(String keyName, String valueName, Object value, RegistryValueKind valueKind)
   bei SafeExamBrowser.Lockdown.FeatureConfigurations.RegistryConfigurations.RegistryConfiguration.TrySet(RegistryDataItem item) in C:\Users\appveyor\projects\seb-win-refactoring-4w22kouryjnrjj4t\SafeExamBrowser.Lockdown\FeatureConfigurations\RegistryConfigurations\RegistryConfiguration.cs:Zeile 229.

2023-02-02 15:43:40.747 [11] - WARNING: [EaseOfAccessConfiguration] Failed to disable feature!
2023-02-02 15:43:40.754 [11] - ERROR: Failed to configure EaseOfAccessConfiguration (00700fcd-eff0-4c3d-b787-e9ea49584096)!
2023-02-02 15:43:40.758 [11] - ERROR: Lockdown was not successful!
```

You may want to try to e.g. set the registry key manually, disable the service component (if it isn't required for your exam) or then make sure that there isn't another software or system configuration interfering with the service's operation.

**Reply by EliaSchmid** [reply]
Ok...

Couple of questions:

What is the registry key / how do I set it manually?

What is the service component and where would I disable it?

I assume Anti-Virus would interfere with the service's operation but I disabled it before downloading and executing the Configuration-File. Also this Laptop that I am using is brand new and doesn't have anything installed on it so I can't imagine what could be interfering with SEB.

Sorry for asking these questions, nobody actually told me how SEB works nor understood SEB enough to help me out.

**Reply by dbuechel** [reply]
> What is the service component and where would I disable it?

I would advise you to try this first: Open the service control panel of Windows, search for "Safe Exam Browser Service" and stop or even disable it. Then try again to start SEB.

> Also this Laptop that I am using is brand new and doesn't have anything installed on it so I can't imagine what could be interfering with SEB.

We can't really verify that without having physical access to the machine itself. If you could make a video of how you try to start SEB and / or provide a list of all applications you have installed on your system, then we might get a step further here.

**Reply by EliaSchmid** [reply]
**I would advise you to try this first: Open the service control panel of Windows, search for "Safe Exam Browser Service" and stop or even disable it. Then try again to start SEB.**

I tried this, but that didn't fix the problem and resulted in the same error as before. I will attach the new service and runtime logs

**We can't really verify that without having physical access to the machine itself. If you could make a video of how you try to start SEB and / or provide a list of all applications you have installed on your system, then we might get a step further here.**

I wasn't able to record it at the last exam but this is roughly what happened:

- I downloaded the configuration file for the exam
- I opened the file
- SEB started up and loaded the session for some time
- At about half the progression bar, the error appeared

Is there some easy way to generate a comprehensive list of applications on a system?

**Reply by EliaSchmid** [reply]
[2023-03-02_15h00m55s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/10919417/2023-03-02_15h00m55s_Runtime.log)
[2023-03-02_15h02m28s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/10919419/2023-03-02_15h02m28s_Runtime.log)
[2023-03-02_15h02m28s_Service.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/10919420/2023-03-02_15h02m28s_Service.log)
[2023-03-02_15h03m30s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/10919421/2023-03-02_15h03m30s_Runtime.log)
[2023-03-02_15h03m53s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/10919422/2023-03-02_15h03m53s_Runtime.log)
[2023-03-02_15h04m31s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/10919423/2023-03-02_15h04m31s_Runtime.log)
[2023-03-02_15h04m31s_Service.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/10919424/2023-03-02_15h04m31s_Service.log)

Here are the files, I tried to open the session a couple of times. I deactivated my antivirus before downloading the configuration file, then disabled the service component, then opened the file

**Reply by omerabdul** [reply]
Hi @dbuechel, I'm trying to manage an exam. Could you tell me if the log files are automatically sent to the server side in real time or is it required for the test-taker to manually share logs after session is closed?

**Reply by dbuechel** [reply]
Please do create a new discussion for topics unrelated to the original question. No, SEB does not communicate with any server applications unless you're e.g. using SEB Server.

**Comment by leo00oel** [comment]
![Screenshot 2025-01-21 190826](https://github.com/user-attachments/assets/ee2060ac-2fe0-4ee2-8068-8c9122903390)
how can i slove this error?

**Reply by dbuechel** [reply]
Please provide the log files.

**Reply by leo00oel** [reply]
[2020-09-13_19h35m54s_Runtime.log](https://github.com/user-attachments/files/18533658/2020-09-13_19h35m54s_Runtime.log)
is this the log files?

**Reply by dbuechel** [reply]
This is one of them revealing that there was an issue with the client component while attempting to initiate a new session:

```
2020-09-13 19:36:17.793 [16] - DEBUG: [Process 'SafeExamBrowser.Client.exe' (10228)] Process has terminated.
2020-09-13 19:36:17.795 [07] - ERROR: Client instance terminated unexpectedly during initialization!
2020-09-13 19:36:17.798 [07] - ERROR: Failed to start new client instance! Aborting procedure...
```

Please provide the client log file specified in the error message above to determine the exact reason of failure.

**Reply by dbuechel** [reply]
Terribly sorry, I just realize that this is not at all the correct log file; you posted a log file from the year 2020! Please do provide _exactly_ the ones specified in the error message above.

**Reply by leo00oel** [reply]
um can u suggest me how to get those log files?

**Reply by dbuechel** [reply]
Open the path specified in the error message in Windows explorer. All log files of SEB 3.x for Windows are always located under `%LocalAppData%\SafeExamBrowser\Logs`.

**Comment by yiemyi05** [comment]
Hola, han podido hallar alguna solución al problema?

**Reply by dbuechel** [reply]
Please specify the exact problem you're encountering resp. provide the log files of the affected session. The session start error message is generic and may have a variety of underlying causes.

**Comment by Simie851** [comment]
Saya mengalami masalah yang sama ini saya lampirkan file client dan runtime saya, terimakasih
[
[2025-04-15_14h00m34s_Client.log](https://github.com/user-attachments/files/19752975/2025-04-15_14h00m34s_Client.log)
[2025-04-15_14h00m34s_Runtime.log](https://github.com/user-attachments/files/19752978/2025-04-15_14h00m34s_Runtime.log)
](url)

**Reply by bayualsadewa** [reply]
apakah anda sudah menemukan cara mengatasinya? aku juga mengalami masalah yang sama.

**Comment by chano260** [comment]
Hi, I installed seb using Windows 11, the installation was fine, but when I open the app, I got an error like this:
![image](https://github.com/user-attachments/assets/73299ad4-d863-400d-b664-23ba76631d23)
I've already uninstalled all antivirus software, turn off windows security but still does not work.
Here is the log files:
[2025-04-18_15h26m02s_Runtime.log](https://github.com/user-attachments/files/19810518/2025-04-18_15h26m02s_Runtime.log)
[2025-04-18_15h26m02s_Client.log](https://github.com/user-attachments/files/19810519/2025-04-18_15h26m02s_Client.log)
How to fix that?

**Comment by JStromGY** [comment]
@Simie851 @chano260 In Windows, go to Settings -> Privacy & Security -> Location and turn on positioning for SEB. After that it should work for both of you.

**Reply by bayualsadewa** [reply]
i doing all this, but is not solve the problem
![WhatsApp Image 2025-05-20 at 14 46 22_e84729e2 (1)](https://github.com/user-attachments/assets/c875a99e-b75a-4771-8d6a-de69b5b2ef14)
[2025-05-20_14h23m58s_Client.log](https://github.com/user-attachments/files/20341667/2025-05-20_14h23m58s_Client.log)
[2025-05-20_14h23m58s_Runtime.log](https://github.com/user-attachments/files/20341689/2025-05-20_14h23m58s_Runtime.log)
Maybe you can help me about my problem. this my log file.

**Reply by JStromGY** [reply]
Can you go into "c:\program files\safeexambrowser\application" and confirm that there is a file called CefSharp.Core.Runtime.dll in that directory?
The logs says it is missing and that's why SEB fails to start.

**Reply by bayualsadewa** [reply]
yes i have the file of CefSharp.Core.Runtime.dll on that folder.
![Screenshot 2025-05-20 170255](https://github.com/user-attachments/assets/db1bfe5d-a42e-44a8-a9c3-a6d0fbe1787e)
i tried anything way like give question to chatgpt, and almost solve the problem. main problem cant solve but in the log file i have new problem, the problem is 
"[Process 'SafeExamBrowser.Client.exe' (7788)] Process has terminated. Client instance terminated unexpectedly during initialization!". 
Until here, chatgpt cant help me again

**Reply by dbuechel** [reply]
@bayualsadewa Please try again with the latest release version; 3.7.0 had, among other things, a concurrency issue which could lead to crashes.

**Reply by bayualsadewa** [reply]
I am use 3.7.1 version and i can installed normally, but if i use to web test, i had the same problem (same error).
if you mean i must use the new version maybe 3.9.0, i cant do it, because the requirement of test from the institute said we must use 3.7.0 version.

**Comment by singhal-amit** [comment]
![Screenshot 2025-06-17 130132](https://github.com/user-attachments/assets/23161da7-6cc2-498f-9beb-358b34912f59)

Hey, I was trying to ready the SEB for the test and got the same error. I checked the log files and the error was given as:

```c++

2025-06-17 21:29:09.147 [01] - ERROR: Caught unexpected exception while performing operation 'LazyInitializationOperation'!

   Exception Message: Access is denied
   Exception Type: System.ComponentModel.Win32Exception

   at SimpleWifi.Win32.WlanClient..ctor()
   at SimpleWifi.Wifi..ctor()
   at SafeExamBrowser.SystemComponents.Network.NetworkAdapter.Initialize() in C:\Users\appveyor\projects\seb-win-refactoring-2cm5n6dku3tpmv82\SafeExamBrowser.SystemComponents\Network\NetworkAdapter.cs:line 92
   at SafeExamBrowser.Client.Operations.ShellOperation.InitializeSystemComponents() in C:\Users\appveyor\projects\seb-win-refactoring-2cm5n6dku3tpmv82\SafeExamBrowser.Client\Operations\ShellOperation.cs:line 230
   at SafeExamBrowser.Client.Operations.ShellOperation.Perform() in C:\Users\appveyor\projects\seb-win-refactoring-2cm5n6dku3tpmv82\SafeExamBrowser.Client\Operations\ShellOperation.cs:line 85
   at SafeExamBrowser.Core.OperationModel.OperationSequence.Perform() in C:\Users\appveyor\projects\seb-win-refactoring-2cm5n6dku3tpmv82\SafeExamBrowser.Core\OperationModel\OperationSequence.cs:line 108

2025-06-17 21:29:09.150 [01] - INFO: Terminating shell...

``` 

I tried disabling my wifi adapter but then it showed No MAC address list was found but my Ethernet and Bluetooth drivers were enabled.

log files -
[2025-06-17_21h29m01s_Client.log](https://github.com/user-attachments/files/20780489/2025-06-17_21h29m01s_Client.log)
[2025-06-17_21h29m01s_Runtime.log](https://github.com/user-attachments/files/20780488/2025-06-17_21h29m01s_Runtime.log)

**Reply by singhal-amit** [reply]
edit: I got it fixed by downloading latest version of SEB 3.9 from the repo. I would still like to know the cause of this issue for the release of 3.7 version.

**Reply by JStromGY** [reply]
> edit: I got it fixed by downloading latest version of SEB 3.9 from the repo. I would still like to know the cause of this issue for the release of 3.7 version.

Hi, it's a change Microsoft did with one of their updates in late autumn/early winter. The update requires applications to ask for permission to use the Positioning functionality in a way that is similar to how it is on phones. For version 3.7(the one you used) and earlier versions this can be fixed by enabling positioning for Safe Exam Browser manually in the Windows Privacy menu. Hope this answers your question!

**Reply by singhal-amit** [reply]
Hey, thanks for the fix you provided. I encountered a version compatibility issue with 3.9, so downloaded 3.7 from the documentation and got the same error. I followed the steps you provided above and it worked. Thanks again.

Simplified Steps for anyone encountering same issue - 

- Step 1: click Start → Settings.
- Step 2: go to Privacy & Security  → “Location” under App permissions.
- Step 3: Turn On Location Services
- Step 4: Turn On “Let apps access your location” and “Let desktop apps access your location”.
- Step 5: Enable for Safe Exam Browser in the list.
(If it’s not visible yet, you may need to open SEB once first — Windows will then detect it as a desktop app and list it here.)
- Step 6: Restart SEB

**Comment by sxtyxm123** [comment]
same issue 
[2025-07-14_00h28m47s_Runtime.log](https://github.com/user-attachments/files/21205842/2025-07-14_00h28m47s_Runtime.log)
[2025-07-14_00h28m47s_Client.log](https://github.com/user-attachments/files/21205843/2025-07-14_00h28m47s_Client.log)
<img width="1366" height="768" alt="image" src="https://github.com/user-attachments/assets/ae434620-4220-4ccb-ae59-c030f91ce567" />

**Reply by dbuechel** [reply]
Verify that you have granted the location permission as already explained in #928.

**Reply by sxtyxm123** [reply]
> Verify that you have granted the location permission as already explained in #928.

thank you so much  for reply but this issue was resolved by my university's  lab technician, yes you are right it was the location access related issue after allowing it was working  but still there was an error but  after that he also configured the SEB and saved my university's LMS link then it was working fine

**Comment by dbuechel** [comment]
This discussion has become too broad resp. concerning many (often unrelated) topics as the error message of the OP above is generic and may have many different causes.

**Comment by shimselv** [comment]
hi, i have similiar problem with SEB, here the issue, i use 3.9 version and can't have older version

[2025-08-06_20h29m45s_Runtime.log](https://github.com/user-attachments/files/21619780/2025-08-06_20h29m45s_Runtime.log)
[2025-08-06_20h29m45s_Client.log](https://github.com/user-attachments/files/21619781/2025-08-06_20h29m45s_Client.log)

is there anything i can do?

**Reply by JStromGY** [reply]
> hi, i have similiar problem with SEB, here the issue, i use 3.9 version and can't have older version
> 
> [2025-08-06_20h29m45s_Runtime.log](https://github.com/user-attachments/files/21619780/2025-08-06_20h29m45s_Runtime.log) [2025-08-06_20h29m45s_Client.log](https://github.com/user-attachments/files/21619781/2025-08-06_20h29m45s_Client.log)
> 
> is there anything i can do?

Hi, the thing that is blocking SEB from starting is TeamViewer, specifically TeamViewer_Service.exe. If you uninstall the program or kill/stop that process then things should work. You can do this by pressing ctrl-shift-escape to open task manager and from there locate it under either the 'processes' or 'services' tab.

**Reply by shimselv** [reply]
Hi, I tried to unistall TeamViewer and the SEB works as usual.
Thank you so much for your advice.

**Comment by duzousolom** [comment]
hello,i got a same issue,please help me check this
[2025-09-21_16h07m50s_Runtime.log](https://github.com/user-attachments/files/22450083/2025-09-21_16h07m50s_Runtime.log)
[2025-09-21_16h07m50s_Service.log](https://github.com/user-attachments/files/22450084/2025-09-21_16h07m50s_Service.log)

**Reply by dbuechel** [reply]
```
2025-09-21 16:07:56.296 [04] - ERROR: [EaseOfAccessConfiguration] Failed to set registry item 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\Utilman.exe\Debugger' => 'SebDummy.exe'!

   Exception Message: 尝试执行未经授权的操作。
   Exception Type: System.UnauthorizedAccessException

   在 Microsoft.Win32.RegistryKey.Win32Error(Int32 errorCode, String str)
   在 Microsoft.Win32.RegistryKey.SetValue(String name, Object value, RegistryValueKind valueKind)
   在 Microsoft.Win32.Registry.SetValue(String keyName, String valueName, Object value, RegistryValueKind valueKind)
   在 SafeExamBrowser.Lockdown.FeatureConfigurations.RegistryConfigurations.RegistryConfiguration.TrySet(RegistryDataItem item) 位置 C:\Users\appveyor\projects\seb-win-refactoring-6tj6ae97ro290h9u\SafeExamBrowser.Lockdown\FeatureConfigurations\RegistryConfigurations\RegistryConfiguration.cs:行号 229
```

You can use the SEB Reset Utility to resolve the issue.

**Reply by JStromGY** [reply]
> ```
> 2025-09-21 16:07:56.296 [04] - ERROR: [EaseOfAccessConfiguration] Failed to set registry item 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows NT\CurrentVersion\Image File Execution Options\Utilman.exe\Debugger' => 'SebDummy.exe'!
> 
>    Exception Message: 尝试执行未经授权的操作。
>    Exception Type: System.UnauthorizedAccessException
> 
>    在 Microsoft.Win32.RegistryKey.Win32Error(Int32 errorCode, String str)
>    在 Microsoft.Win32.RegistryKey.SetValue(String name, Object value, RegistryValueKind valueKind)
>    在 Microsoft.Win32.Registry.SetValue(String keyName, String valueName, Object value, RegistryValueKind valueKind)
>    在 SafeExamBrowser.Lockdown.FeatureConfigurations.RegistryConfigurations.RegistryConfiguration.TrySet(RegistryDataItem item) 位置 C:\Users\appveyor\projects\seb-win-refactoring-6tj6ae97ro290h9u\SafeExamBrowser.Lockdown\FeatureConfigurations\RegistryConfigurations\RegistryConfiguration.cs:行号 229
> ```
> 
> You can use the SEB Reset Utility to resolve the issue.

What options do you have if SEB Reset Utility can't be used?
Our scenario: Needs admin to run, running it with an admin-account runs the reset on the wrong account.

**Reply by dbuechel** [reply]
Use the option to reset to default values, then enter the correct user name; or you delete the key manually in the registry editor.

**Comment by Kashish23092004** [comment]
<img width="1071" height="754" alt="image" src="https://github.com/user-attachments/assets/dfa5f62b-c5db-429a-88f0-1bb4cc7fe4ef" />
I am getting an error in my Safe Exam Browser and it stops working before my exam. I have my exam today and it’s urgent. Please see the attached screenshot of the problem. Can you suggest how to fix it quickly

**Reply by dbuechel** [reply]
You'll need to provide the log file specified in the error message.

**Comment by 12pradeep7m** [comment]
i also same issue pls help what to do next?
![46b5a8bf-36c8-43ef-87b6-d7f8a6ab9a48](https://github.com/user-attachments/assets/7e57421f-3991-4ef6-9174-425199a7d58b)
[2025-11-13_16h15m43s_Service.log](https://github.com/user-attachments/files/23545793/2025-11-13_16h15m43s_Service.log)

**Reply by dbuechel** [reply]
Again, you're using a custom build.

**Comment by 12pradeep7m** [comment]
[2025-11-13_16h15m43s_Runtime.log](https://github.com/user-attachments/files/23545807/2025-11-13_16h15m43s_Runtime.log)
[2025-11-13_16h15m43s_Service.log](https://github.com/user-attachments/files/23545811/2025-11-13_16h15m43s_Service.log)
